app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
```

Comment:
Please provide type annotation here

Reply:
I hope I did the annotation of the type correctly in next commit))))